mmv1/third_party/terraform/website/docs/r/sql_database_instance.html.markdown

```diff
@@ -572,6 +579,38 @@ performing filtering in a Terraform config.
 
 * `server_ca_cert.0.sha1_fingerprint` - SHA Fingerprint of the CA Cert.
 
+## Switchover (SQL Server Only)
+Users can perform a switchover on any direct `cascadable` replica by following the steps below.
+
+  ~>**WARNING:** Failure to follow these steps can lead to data loss (You will be warned during plan stage). To prevent data loss during a switchover, please verify your plan with the checklist below.
+
+For a more in-depth walkthrough with example code, see the [Switchover Guide](../guides/sql_instance_switchover.html.markdown)
```

Comment:
Is it helpful to include the rest of the steps here versus just directing people to the full guide?

Reply:
All the 3 steps to invoke switchover are already included below, the full guide has the additional example code